tests/hf/test_spin.py

```diff
@@ -0,0 +1,240 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for functions needed for computing the spin observables.
+"""
+import pennylane as qml
+import pytest
+from pennylane import Identity, PauliX, PauliY, PauliZ
+from pennylane import numpy as np
+from pennylane.hf.spin import _spin2_matrix_elements, spin2, spinz
+
+
+@pytest.mark.parametrize(
```

Comment:
I guess this was computed by hand?

Reply:
Yes, these tests are copied from PL-QChem.